examples/Consumer/Program.cs

```diff
@@ -160,12 +149,12 @@ public static void Run_ManualAssign(string brokerList, List<string> topics, Canc
             {
                 // the group.id property must be specified when creating a consumer, even 
                 // if you do not intend to use any consumer group functionality.
-                GroupId = Guid.NewGuid().ToString(),
+                GroupId = "csharp-consumer-GROUPID-NOT-USED",
```

Comment:
maybe: "groupid-not-used-but-mandatory" ?

Reply:
Updated.